qa/L0_infer/test.sh

```diff
@@ -35,6 +35,9 @@ if [ -z "$REPO_VERSION" ]; then
     exit 1
 fi
 
+# Install pytorch
+pip3 install torch==1.6.0+cpu torchvision==0.7.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Why do we need pytorch to run L0_infer? We can't install pytorch here as it may interfere with the existing libtorch testing happening in this test.

Reply:
I'd have a separate L0_infer_python test for the python scripts that installs the necessary packages. Additionally why do you need torchvision for any of your tests? Don't think you use torchvision ops in these addsub models

Reply:
I've removed Pytorch from all the installations and simplified AddSub so that it doesn't require PyTorch and uses only numpy.